kDrive/UI/Controller/Files/FileActionsFloatingPanelViewController.swift

```diff
@@ -24,6 +24,31 @@ import kDriveResources
 import LinkPresentation
 import UIKit
 
+public enum FloatingPanelActionIdentifier: Int {
```

Comment:
We could use an enum with no raw value and have `FloatingPanelAction.id: FloatingPanelActionIdentifier` to prevent passing simply an Int. 

What do you think ?

```suggestion
public enum FloatingPanelActionIdentifier {
```

Reply:
I went for a minimum change / max impact kind of change, thus no change on `FloatingPanelAction`.

Yeah this does removes a class of issues dealing with `int` directly when switching on this.